engine/modelc/src/modelimporter_gltf.cpp

```diff
@@ -379,29 +379,30 @@ static char* CreateNameFromHash(const char* prefix, uint32_t hash)
     return strdup(buffer);
 }
 
-template <typename T>
-static char* CreateObjectName(T* object, const char* prefix, uint32_t index)
+static char* CreateCgltfName(const char* prefix, uint32_t index)
```

Comment:
It just became too cumbersome to check multiple places for strings.
So, we now make sure that all cgltf objects have valid names.